maintainers/scripts/pluginupdate.py

```diff
@@ -460,6 +460,11 @@ def parse_args(editor: Editor):
         default=30,
         help="Number of concurrent processes to spawn.",
     )
+    parser.add_argument(
+        "--no-commit", "-n", action="store_false", default=True,
```

Comment:
```suggestion
        "--no-commit",
        "-n",
        action="store_false",
        default=False,
```

Reply:
I added an intermediate variable to make it more readable.